src/sentry/static/sentry/app/components/group/externalIssueActions.tsx

```diff
@@ -48,13 +60,13 @@ class ExternalIssueActions extends AsyncComponent {
     }/?externalIssue=${issueId}`;
     this.api.request(endpoint, {
       method: 'DELETE',
-      success: (data, _, jqXHR) => {
+      success: (_data, _, _jqXHR) => {
```

Comment:
@nolachen If the arguments are not used, why not just remove them from the callback?